[2025-04-24T02:05:24.622+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-24T02:05:24.653+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: build_crypto_dbt.dbt_test manual__2025-04-24T02:05:10.728326+00:00 [queued]>
[2025-04-24T02:05:24.680+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: build_crypto_dbt.dbt_test manual__2025-04-24T02:05:10.728326+00:00 [queued]>
[2025-04-24T02:05:24.681+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-24T02:05:24.710+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): dbt_test> on 2025-04-24 02:05:10.728326+00:00
[2025-04-24T02:05:24.733+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'build_crypto_dbt', 'dbt_test', 'manual__2025-04-24T02:05:10.728326+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/build_crypto_dbt.py', '--cfg-path', '/tmp/tmpvgga2rh7']
[2025-04-24T02:05:24.739+0000] {standard_task_runner.py:105} INFO - Job 26: Subtask dbt_test
[2025-04-24T02:05:24.740+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=355) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-24T02:05:24.744+0000] {standard_task_runner.py:72} INFO - Started process 356 to run task
[2025-04-24T02:05:24.813+0000] {task_command.py:467} INFO - Running <TaskInstance: build_crypto_dbt.dbt_test manual__2025-04-24T02:05:10.728326+00:00 [running]> on host 09ab332ca4a7
[2025-04-24T02:05:24.915+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='build_crypto_dbt' AIRFLOW_CTX_TASK_ID='dbt_test' AIRFLOW_CTX_EXECUTION_DATE='2025-04-24T02:05:10.728326+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-24T02:05:10.728326+00:00'
[2025-04-24T02:05:24.916+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-24T02:05:24.935+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-24T02:05:24.938+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/***/.local/bin/dbt test --profiles-dir /opt/***/build_crypto --project-dir /opt/***/build_crypto']
[2025-04-24T02:05:24.950+0000] {subprocess.py:86} INFO - Output:
[2025-04-24T02:05:27.566+0000] {subprocess.py:93} INFO - [0m02:05:27  Running with dbt=1.5.5
[2025-04-24T02:05:28.372+0000] {subprocess.py:93} INFO - [0m02:05:28  Registered adapter: snowflake=1.5.5
[2025-04-24T02:05:28.610+0000] {subprocess.py:93} INFO - [0m02:05:28  Found 3 models, 0 tests, 1 snapshot, 0 analyses, 322 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics, 0 groups
[2025-04-24T02:05:28.612+0000] {subprocess.py:93} INFO - [0m02:05:28
[2025-04-24T02:05:28.613+0000] {subprocess.py:93} INFO - [0m02:05:28  Nothing to do. Try checking your model configs and model specification args
[2025-04-24T02:05:29.393+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-24T02:05:29.431+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-24T02:05:29.432+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=build_crypto_dbt, task_id=dbt_test, run_id=manual__2025-04-24T02:05:10.728326+00:00, execution_date=20250424T020510, start_date=20250424T020524, end_date=20250424T020529
[2025-04-24T02:05:29.457+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-24T02:05:29.492+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-24T02:05:29.494+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
